state-chain/pallets/cf-pools/src/lib.rs

```diff
@@ -971,10 +973,14 @@ pub mod pallet {
 		#[pallet::weight(T::WeightInfo::set_maximum_price_impact())]
 		pub fn set_maximum_price_impact(
 			origin: OriginFor<T>,
+			base_asset: Asset,
+			quote_asset: Asset,
```

Comment:
Can we make it `Vec<(Asset, Asset)>` or just `Vec<Asset>` (since we know the quote asset is always `STABLE_ASSET`)? 

I can imagine we might want to update these in batches rather than one at a time.

Reply:
I see. Understood

Reply:
Sorry I didn't explain very well - I should have said `Vec<(Asset, Tick)>` so we can update multiple pools with one governance call.